iocAdmin/src/iocReleaseCreateDb.py

```diff
@@ -47,19 +47,25 @@ def export_db_file( module_versions, path=None):
         x = key.replace("_MODULE_VERSION","",1)
         if idx >= idxMax:
             break
-        print('record(stringin, "$(IOC):RELEASE%02d") {' % idx, file=out_file)
+        print('record(%s, "$(IOC):RELEASE%02d") {' % ('lsi' if is_epics7 else 'stringin', idx), file=out_file)
+        if is_epics7:
+            print('  field(INP, {const:"%s"})' % module_version, file=out_file)
+        else:
+            print('  field(VAL, "%s")' % (module_version[:38] if not is_epics7 else module_version), file=out_file) # Truncate module_version at 40 chars (including NULL terminator). Otherwise this DB will fail to load.
```

Comment:
`(module_version[:38] if not is_epics7 else module_version)`

is_epics7 will always be False here after the "else:" statement.

Reply:
Oops, good catch!